hazelcast/src/main/java/com/hazelcast/sql/impl/expression/datetime/ToTimestampTzFunction.java

```diff
@@ -74,30 +74,47 @@ public QueryDataType getType() {
     }
 
     /**
-     * Returns TemporalUnit of the given unixTimestamp, using the magnitude of the value to determine most likely unit.
-     * - unixTimestamp < MILLISECONDS_IN_YEAR (365 * 86400 * 1000) - timestamp is in Seconds
-     * - unixTimestamp >= MILLISECONDS_IN_YEAR (365 * 86400 * 1000) - timestamp in in Milliseconds
-     * - unixTimestamp >= MICROSECONDS_IN_YEAR (365 * 86400 * 1000_000) - timestamp is in Microseconds
-     * - unixTimestamp >= NANOSECONDS_IN_YEAR (365 * 86400 * 1000_000_000) - timestamp is in Nanoseconds
-     * - unixTimestamp < 0 - timestamp is in Seconds
+     * Returns TemporalUnit of the given unixTimestamp, using the
+     * magnitude of the value to determine most likely unit.
+     * <ol>
+     * <li>unixTimestamp < MILLISECONDS_IN_YEAR (365 * 86400 *
+     * 1000) - timestamp is in seconds</li>
+     * <li>unixTimestamp >= MILLISECONDS_IN_YEAR (365 * 86400 * 1000)
+     * - timestamp is in milliseconds</li>
+     * <li>unixTimestamp >= MICROSECONDS_IN_YEAR
+     * (365 * 86400 * 1000_000) - timestamp is in microseconds</li>
+     * <li>unixTimestamp >= NANOSECONDS_IN_YEAR (365 * 86400
+     * * 1000_000_000) - timestamp is in nanoseconds</li>
+     * <li>unixTimestamp < 0 - timestamp is in seconds</li>
+     * </ol>
      *
-     * Note that this also imposes limits on the possible max correctly interpreted timestamp value,
-     * that would be otherwise correctly interpreted as in Seconds/Milliseconds/Microseconds.
-     * For example if this function is called on a unixTimestamp of 31_536_000_001 which in seconds
-     * corresponds to datetime of 2969-05-03 00:00:01, this function would instead interpret it
-     * as 1971-01-01 00:00:00.001. This is also true for maximum date expressed in milliseconds and microseconds.
-     * Additionally this of course also imposes the min date possible to be interpreted in Milliseconds,
-     * Microseconds and Nanoseconds.
+     * <p>Note that this also imposes limits on the possible max
```

Comment:
The starting `<p>` should be on a separate line:
```suggestion
     * <p>
     * Note that this also imposes limits on the possible max
```